src/native/file-system.js

```diff
@@ -0,0 +1,28 @@
+const fs = require('fs');
+
+function fileExists(path) {
```

Comment:
Thinking we should change all these functions to be async so that everything can load together.

Reply:
Agreed and completed. :)